zebROS_ws/src/tf_object_detection/src/config/model_ssd_mobilenet_v2_512x512.py

```diff
@@ -0,0 +1,88 @@
+import graphsurgeon as gs
+import tensorflow as tf
+
+name = 'ssd_mobilenet_v2_512x512'
+path = '/home/ubuntu/tensorflow_workspace/2020Game/models/' + name + '.pb'
```

Comment:
This should be updated to pick up something from the 2020Offseason repo. tensorflow_workspace typically isn't installed on the robot, just on computers we use for labeling and training images.